src/index.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
```

Comment:
Please avoid committing .js files in typescript projects. You can easily avoid this by making typescript put the build in a separate folder and the ignore that folder in git. To do so use the `outDir` property in `tsconfig.json` and the `.gitignore` file in the project root